### ISSUE TITLE
TypeError. reading uriParameters. CRITICAL

### ISSUE THREAD

**Issue by menduz** [raml-org/raml-js-parser-2#370]
Hi, I'm trying to do this:

```
resource.absoluteUriParameters().map(x => x.toJSON());
```

on the first resource of this RAML:

```raml
#%RAML 0.8
title: raml-service
version: 0.1
mediaType: application/json

baseUri: http://123

/echo:
  get:
    description: |
      Get status of RAML service
    responses:
      200:
        body:
          application/json:
            example: |
              { "status": "up" }
```

and throws:
```
TypeError: Cannot read property 'value' of null
    at Object.absoluteUriParameters (/Users/agustinmendez/Code/http-bat/node_modules/raml-1-parser/dist/raml1/wrapped-ast/wrapperHelper08.js:397:32)
    at ResourceImpl.absoluteUriParameters (/Users/agustinmendez/Code/http-bat/node_modules/raml-1-parser/dist/raml1/artifacts/raml08parser.js:2301:23)
```

It does **not fail when you specify the `baseUri`**

**Comment by menduz** [comment]
Thanks @ddenisenko

**Comment by ddenisenko** [comment]
Fixed. The fix will be published in 0.8.29.
